DigitalLearningSolutions.Web.Tests/Controllers/LearningMenu/LearningMenuControllerTests.cs

```diff
@@ -430,6 +430,300 @@ public void Sections_should_render_view()
                 .Model.Should().BeEquivalentTo(expectedModel);
         }
 
+        [Test]
+        public void Sections_should_redirect_to_section_page_if_one_tutorial_in_section()

```

Comment:
Should redirect to tutorial page? 🙂

Reply:
Yes, sorry those were named wrong! Fixed in 0a65197 👍